docs/src/radiative_transfer.md

```diff
@@ -99,9 +98,25 @@ The calculation accounts for:
 - Hour angle (based on solar time)
 - Latitude (for observer position)
 
+## Clear-sky Full-spectrum Radiation
+
+For more accurate radiative transfer calculations, use the `:clear_sky` optics flavor which computes full-spectrum gas optics using RRTMGP's lookup tables:
```

Comment:
How about using a singleton type `<: AbstractOptics`, instead of `Val{<:Symbol}`?  At least the signature is more "meaningful".

Reply:
Hmm yeah, I think that would work just fine.

Reply:
I'll make that change